### ISSUE TITLE
JavaScript interactive examples are keyboard traps

### ISSUE THREAD

**Issue by bsmth** [mdn/yari#12712]
### Summary

>I was tabbing through [a page on MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/try...catch#try_it), and I found that once I tabbed _into_ one of the "Try It" code blocks, I could not tab out. Hitting tab indents the code in the code block, and shift+tab removes the indent.
> 
> Naturally, this is bad for accessibility as it means that keyboard users cannot continue to use the page. 
> 
> I would recommend removing the ability to indent code with tab and reserving tab for changing focus. See [WCAG](https://www.w3.org/TR/WCAG22/#no-keyboard-trap) for more details.

Copied over from https://github.com/orgs/mdn/discussions/743

### URL

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/try...catch#try_it

### Reproduction steps

1. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/try...catch#try_it
2. <kbd>Tab</kbd> a few times

### Expected behavior

you can focus on the next element in the page

### Actual behavior

You're stuck in the code editor

### Device

Desktop

### Browser

Firefox

### Browser version

Stable

### Operating system

Mac OS

### Validations

- [x] I have read the [Community Participation Guidelines](https://www.mozilla.org/en-US/about/governance/policies/participation/).
- [x] I have verified that there isn't [already an issue](https://github.com/mdn/yari/issues) that reports the same bug to avoid creating a duplicate.
- [x] I have checked that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/orgs/mdn/discussions/categories/platform).

**Comment by bsmth** [comment]
CCing @stevegoldberg who was the original reporter.

**Comment by caugner** [comment]
I verified [via Web Archive](https://web.archive.org/web/20240223073801/https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/try...catch) that this does not seem to be a regression caused by the new interactive examples architecture.

For the record, it seems to be possible to leave the editor by typing <kbd>Esc</kbd> and <kbd>Shift</kbd> + <kbd>Tab</kbd>.

I wonder what the best solution would be. Should tab focus the container (not the editor), and enter focus the editor, and escape focus the container again?

**Comment by bsmth** [comment]
No I don't think this is a regression.

> Should tab focus the container (not the editor), and enter focus the editor, and escape focus the container again?

This makes sense to me, at least, curious to hear other approaches.

**Comment by caugner** [comment]
Thanks for bringing this up, Miruna. @fiji-flo suggests to add a small message below the editor explaining how to leave the editor.
